zebra-scan/src/storage/db/sapling.rs

```diff
@@ -95,37 +95,39 @@ impl Storage {
             .collect()
     }
 
-    /// Returns all the keys and their birthday heights.
-    pub fn sapling_keys_and_birthday_heights(&self) -> HashMap<SaplingScanningKey, Height> {
-        // This code is a bit complex because we don't have a separate column family for keys
-        // and their birthday heights.
-        //
-        // TODO: make a separate column family after the MVP.
-
+    /// Returns all the keys and their last scanned heights.
+    pub fn sapling_keys_and_last_scanned_heights(&self) -> HashMap<SaplingScanningKey, Height> {
```

Comment:
In a new PR we can add a test that compares `sapling_results_for_key(key).keys().max()` with `sapling_keys_and_last_scanned_heights().get(key)`, and makes sure they match.

Reply:
I have code for this now, I'll push it when it works.